app/Controllers/feedController.php

```diff
@@ -24,6 +24,7 @@ public function firstAction() {
 				Minz_Error::error(403);
 			}
 		}
+		$this->updateTTL();
```

Comment:
We need to call that from a few other places. I suggest:
* In `install.php` for people going through it after an upgrade (a bit redundant, but the earlier the better);
* In `FreshRSS_FeedDAO->listFeedsOrderUpdate()` to cover the CLI and the JavaScript manual refresh;

Reply:
I've done it for the CLI and JavaScript, but after looking into it, I think it's a lot of work to do it during installation. I am not sure it's that important.

Reply:
I think it would be good to add an `updateTTL()` in `FreshRSS_subscription_Controller::firstAction()`.
In my case, I am seeing:
![image](https://user-images.githubusercontent.com/1008324/34469491-71e1591e-ef20-11e7-8d8b-8fc4c72cdbc7.png)

Reply:
Done, but not tested.